src/auth.py

```diff
@@ -97,8 +99,14 @@ def require_admin_status(func):
 	@wraps(func)
 	def wrapper(*args, **kwargs):
 		netid = kwargs.get(UID_KEY, None)
-		if request.json is not None:
-			netid = request.json.get(UID_KEY, netid)
+		if netid is None and request.json is not None:
+			_json = request.json
+			if isinstance(_json, str):
```

Comment:
If the mimetype is set correctly, shouldn't this always be either a dictionary or a list? Why is it necessary to check if the json is a string?

Reply:
I believe this was an error on the cli's side that I fixed by setting the cli appropriately, and I just never omitted it because it never got triggered again.